daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Result.scala

```diff
@@ -41,7 +41,7 @@ sealed trait Result[+A] extends Product with Serializable {
   }
 
   def consume(
-      pcs: ContractId => Option[VersionedContractInstance],
+      pcs: ContractId => Option[ContractInstance],
```

Comment:
Having the `VersionedContractInstance` is a bit of a pain in speedy because we need to get the version for the disclosed contracts somehow. This is particularly annoying because we end up throwing the version away. I was hoping this wouldn't be too much of a change for the periphery, but I could also try just adding a dummy version to the disclosed contracts.